docs/content/menu/index.md

```diff
@@ -57,6 +57,7 @@ headless: true
     - [Custom Gradle Plugins]({{< ref "/ci/CustomGradlePlugins.md" >}})
     - [Impact analysis]({{< ref "/ci/ImpactAnalysis.md" >}})
     - [Containers]({{< ref "/ci/Containers.md" >}})
+    - [Dependencies cache]({{< ref "/ci/DependenciesCache.md" >}})
```

Comment:
This menu becomes a blog :thinking:

Reply:
Maybe we should assemble this file automatically